test/test_transforms_v2_refactored.py

```diff
@@ -1899,6 +1933,56 @@ def test_errors_warnings(self, make_input):
         assert out["mask"].dtype == mask_dtype
 
 
+class TestAdjustBrightness:
```

Comment:
I wanted to make sure that we have proper tests for this. This change plus the ones from `test/transforms_v2_{kernel,dispatcher}_infos.py` are standalone. Let me know if you want me to factor them out into a separate PR: